packages/client/src/forms/types/formSchema.types.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @typedef {object} FormSchema
```

Comment:
No need for @typedefs when you are defining a typescript type with the same name. better-docs will generate the jsdoc from the typescript type automatically.